include/deal.II/grid/grid_reordering.h

```diff
@@ -35,7 +35,7 @@ DEAL_II_NAMESPACE_OPEN
  * this class. Usage of the old-style numbering is deprecated.
  */
 template <int dim, int spacedim = dim>
-class DEAL_II_DEPRECATED GridReordering
+class GridReordering
```

Comment:
?? Do we use this class anywhere so that we want to un-deprecate it?

Reply:
Ah, forgot to hit send:
All its members are deprecated so this only avoids deprecation warnings when defining those in the `*.cc` file.